app/javascript/components/App.js

```diff
@@ -1,27 +1,27 @@
-import React, { Component } from 'react'
-import PropTypes from "prop-types"
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
-class App extends React.Component {
+import React, { Component } from "react"
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom'
+import Header from './components/Header'
+import Home from './pages/Home'
+import './App.css'
+
 
+class App extends React.Component {
```

Comment:
Since you destructured Component on line 1 you can just extend Component rather than React.Component